src/hooks/contracts/useVaultContract.ts

```diff
@@ -29,6 +30,11 @@ export const useVaultContract = () => {
     return receipt.transactionHash;
   };
 
+  const totalAssetsLocked = async (vault: VaultInfo, provider: Provider) => {
```

Comment:
importing `Provider` is unnecessary I think because ethers already has the providers class:
```ts
const totalAssetsLocked = async (vault: VaultInfo, provider: ethers.providers.Provider) => etc...
```

Reply:
or you could export a `ProviderOrSigner` type from `types/general.ts` which will allow for a signer or provider to be passed in:
```ts
export type ProviderOrSigner = ethers.providers.Provider | ethers.Signer;
```

Reply:
Ethers wasn't let me import it directly but I think using  a `ProviderOrSigner` type from types/general.ts is better anyways. Cheers

Reply:
Wait I was incorrect, I am just using ethers.providers.Provider now